### PR TITLE
fix: only include make targets in binary

### PR DESCRIPTION
**Pull request by viktorvoltaire**

### CODE REVIEW

File: sg/initfile.go [hunk 1]
```diff
@@ -33,74 +37,78 @@ func generateInitFile(g *codegen.File, pkg *doc.Package, mks []Makefile) error {
...
+							}
+							i++
+						}
+					}
+					ns := "{}."
+					for _, mk := range mks {
+						if mk.namespaceName() != "" && mk.namespaceName() == function.Recv {
+							ns = getNamespaceMetadata(mk)
+						}
+					}
```
Comment (alethenorio) [inline sg/initfile.go:88]: Should this be a loop at all? We are already inside an mk so now we are looping twice. Maybe

```suggestion
		if mk.namespaceName() != "" {
			ns = getNamespaceMetadata(mk)
		}
```

File: sg/initfile.go [hunk 2]
```diff
@@ -33,74 +37,78 @@ func generateInitFile(g *codegen.File, pkg *doc.Package, mks []Makefile) error {
...
+					g.P("if err != nil {")
+					g.P("logger.Fatal(err)")
+					g.P("}")
+				} else {
+					ns := "{}."
+					for _, mk := range mks {
+						if mk.namespaceName() != "" && mk.namespaceName() == function.Recv {
+							ns = getNamespaceMetadata(mk)
+						}
+					}
```
Comment (alethenorio) [inline sg/initfile.go:105]:
```suggestion
		if mk.namespaceName() != "" {
			ns = getNamespaceMetadata(mk)
		}
```

**Review by alethenorio — APPROVED** [review]
We should probably add some generated example targets. These can get difficult to review for correctness :D

File: sg/initfile.go (hunk 1 above)
Comment (viktorvoltaire) [reply sg/initfile.go]: I refactored this before I saw your comment :)

File: sg/initfile.go (hunk 2 above)
Comment (viktorvoltaire) [reply sg/initfile.go]: I refactored this before I saw your comment :)

**Pull request merged**